recovery/elliptics_recovery/iterator.py

```diff
@@ -379,6 +386,19 @@ def iterate_with_stats(cls, node, eid, timestamp_range,
         return result, result_len
 
 
+class CopyIterator(Iterator):
+    def __init__(self, *args,  **kwargs):
+        super(CopyIterator, self).__init__(*args, **kwargs)
+
+    def _start_iterator(self, eid, ranges, itype, flags, timestamp_range):
+        flags |= elliptics.iterator_flags.move
+        return self.session.start_copy_iterator(eid, ranges, [eid.group_id], flags, timestamp_range[0], timestamp_range[1])
```

Comment:
CopyIterator is used in merge recovery only, that's why exactly one group specified here.